src/main/java/graphql/ExecutionInput.java

```diff
@@ -24,7 +25,7 @@ public class ExecutionInput {
     private final GraphQLContext graphQLContext;
     private final Object localContext;
     private final Object root;
-    private final Map<String, Object> variables;
+    private final RawVariables rawVariables;
```

Comment:
That's a good point. This bugfix is tricky because it depended on the variables refactor. I figured it would be easier to cherry pick the refactor as well, rather than try crowbar the bugfix in alone